lm_eval/models/vllm_causallms.py

```diff
@@ -21,10 +21,12 @@
 try:
     import ray
     from vllm import LLM, SamplingParams
+    from vllm.lora.request import LoRARequest
```

Comment:
`LoRARequest` was added in vLLM v0.3.0 and supported through current version v0.4.1

Reply:
Would it be possible to make the import of LoRARequest conditional on a vLLM version check then? To allow users to still use older vLLM versions in case that's desired?

Reply:
Yes. I just added this, let me know what you think.

Reply:
looks good, assuming you've tested this!

Reply:
Yes, I have